package.json

```diff
@@ -25,7 +25,7 @@
     "fh-cls-mongoose": "2.1.1",
     "fh-gridfs": "1.1.1",
     "fh-logger": "0.5.0",
-    "fh-mbaas-client": "0.15.0",
+    "fh-mbaas-client": "github:grdryn/fh-mbaas-client#process-notStats",
```

Comment:
I'll update this when https://github.com/feedhenry/fh-mbaas-client/pull/29 is merged and published